portfolio/src/main/webapp/requests/queries/index.js

```diff
@@ -78,6 +240,52 @@ const loadMoreProjects =  async () => {
     projects.scrollTo({ top: projects.scrollHeight, behavior: "smooth" });
 };
 
+const loadMap = () => {
+    const map = new google.maps.Map(document.getElementById('map'), {
+        center: { lat: 25.655022, lng: -100.274593 },
+        zoom: 13,
+        styles: [...mapStyles]
+    });
+
+    const places = [
```

Comment:
Nice! Just a thing to be cautious about, I don't know the geographic spread of these locations well, but I'd be careful to avoid putting too many near your home address. You probably want to avoid me being able to geolocate your home address within a few miles based off your favorite places.

If they're all near your university, then that should be fine since there's tons of students in that area.

Reply:
Yeah I guess I am exposing myself right hahahah.

But yeah, everything is around the university.